.github/workflows/api-rds-apply.yml

```diff
@@ -0,0 +1,42 @@
+name: api-rds apply terraform
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/api-rds-apply.yml
+      - terraform/services/api-rds/**
+  workflow_dispatch: # Allow manual trigger
+
+jobs:
+  terraform-apply:
+    permissions:
+      contents: read
+      id-token: write
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: ./terraform/services/api-rds
+    strategy:
+      fail-fast: false
+      matrix:
+        app: [ab2d]
+        env: [dev, test, sbx, prod]
```

Comment:
Let's remove all but the test environment for now to only apply there with this merge. That should resolve PLT-691, and we can add the others back in in a future PR after validating in that environment.